### PR TITLE
Updating the README for developers to better explain that JAVA_HOME 

### PR DESCRIPTION
**Pull request by radumereuta**
environment variable is required to point to the JDK installation directory.
Also specify that git needs to run in the command line. The GUI is not enough.

This is in response to issues #365 and #366

### CODE REVIEW

**Comment by traiansf** [comment]
looks good to me

File: src/README [hunk 1]
```diff
@@ -17,7 +17,14 @@ Linux and Mac:
...
+    installation instructions from the 'INSTALL' file.
+
+Ant usually requires setting an environment variable 'JAVA_HOME' pointing
+to the installation directory of the JDK (not to be mistaken with JRE).
+	
+2.3 Git - command line
+Having a GUI client is not enough. Most distributions have an installation
+option to make git accessible in the command line too.
 
 You can test if it works by calling 'ant' in a Terminal.
```
Comment (traiansf) [inline src/README:29]: This line should have been above section 2.3 and maybe you need a similar line for git here

Comment (radumereuta) [inline src/README:29]: Actually no. That was the problem reported in issue #366. If you don't have git installed as a command line tool, ant will still fail.

**Comment by dwightguth** [comment]
Grigore is right, we shouldn't depend on git like this. It's not good practice. If you want to close out issue #366, you need to actually do a check on whether git is in the path from within build.xml.

**Comment by radumereuta** [comment]
@dwightguth is it ready for merge now?
If yes, please go ahead.

**Pull request merged**